### DISCUSSION TITLE
api docs are anemic

### DISCUSSION THREAD

**Discussion by Hadakajime** [getgauge/taiko#2144] [Development]
API docs are missing a lot of information.

For example, there is no comprehensive reference for valid properties passed to `textBox()` for use in selection, such as `{placeholder: "enter text"}`

https://docs.taiko.dev/api/textbox/

and I can't find any reference via the searchbox for `.value()`

also, your api list is not sorted alphabetically:

https://docs.taiko.dev/api/reference/

**Comment by zabil** [comment]
Thanks for the feedback.

`{placeholder: "enter text"}` is implied at https://docs.taiko.dev/api/textbox/ under labelOrAttrValuePairs as placeholder is an attribute. Please feel free to send a PR if you feel it's best explained in the example.

`.value()` is a bit confusing bit it's there if you follow https://docs.taiko.dev/api/textboxwrapper/ https://docs.taiko.dev/api/valuewrapper/ again an example could make it easier.

You can add examples here

https://github.com/getgauge/taiko/blob/329f1a4252c4c0e9fae1728cd42ccae759b6e4f7/lib/taiko.js#L1789-L1805
